internal/webservice/routes.go

```diff
@@ -204,5 +353,39 @@ func (ws *WebService) serveSPA(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	http.FileServer(http.Dir(ws.assetsDir)).ServeHTTP(w, r)
+	http.FileServer(http.Dir(ws.config.AssetsDir)).ServeHTTP(w, r)
 }
+
+func (ws *WebService) serveSwaggerUI(w http.ResponseWriter, r *http.Request) {
+	buf := bytes.NewReader([]byte(SwaggerUIHTML))
+	http.ServeContent(w, r, "index.html", startupTime, buf)
+}
+
+const SwaggerUIHTML = `<!DOCTYPE html>
```

Comment:
Might as well keep it in a file and use go:embed

Reply:
I thought so but didn't want to complicate stuff for a simple page that probably doesn't need maintenance and will likely not be modified soon